components/signature/overview.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Overview
+page_title: Signature Overview
+description: Overview of the Signature component for Blazor.
+slug: signature-overview
+tags: telerik,blazor,signature,overview
+published: True
+position: 0
+---
+
+# Blazor Signature Overview
+
+This article provides information about the <a href = "https://www.telerik.com/blazor-ui/signature" target="_blank">Blazor Signature component</a> and its core features.
+
+The Blazor Signature component provides an area where users can draw their signatures with the mouse, pen, or with their fingertips. You can use the component as part of any application where the users must sign a document. The Blazor Signature can be easily integrated with the [TelerikForm.]({%slug form-overview%}) The component stores the value of the signature as a <a href="https://learn.microsoft.com/en-us/dotnet/api/system.buffers.text.base64?view=net-6.0">base64 string</a>
+
+## Creating Blazor Signature
+
+1. Use the `TelerikSignature` tag to add the component to your razor page.
+1. Set the `Value` (one-way or two-way binding) parameter to a `string` property.
```

Comment:
Breaking phrases with brackets affects readability (value parameter is a single "object" in the sentence). In general, brackets are not that common in English.

```suggestion
1. Set the `Value` parameter to a `string` property. It supports one-way and two-way binding.
```